Run.java

```diff
@@ -23,8 +23,18 @@
  * @author Jody Paul
  * @version 1.8.2
  */
-public class Run {
-    public static void main(String[] args) {
+public final class Run {
+    /**
+     * Default constructor - not used.
```

Comment:
One you add an explicit constructor, Java no longer creates a "default constructor".
Perhaps a more appropriate comment would be "Disable the default constructor for this utility class."